particles/Layer.hpp

```diff
@@ -1,11 +0,0 @@
-#pragma once
```

Comment:
I removed this because this way we can get an automatic UI which will show all the controls